python/usb_console/configs/fc_and_adcs.json

```diff
@@ -0,0 +1,15 @@
+{
+    "seed": 2,
+    "devices": [
+        {
+            "name": "FlightController",
+            "port": "/dev/cu.usbmodem47349801",
+            "stop_if_unconnected": true
+        },
+        {
+            "name": "ADCSController",
+            "port": "/dev/cu.usbmodem",
+            "stop_if_unconnected": true
+        }
+    ]
+}
```

Comment:
Are these configs generated or no? Just curious because everyone's port(s) will most likely be different.

Reply:
These are not autogenerated, but I put them here so that they're available as templates. They're easy to reconfigure.